modules/components/pages/processors/http.adoc

```diff
@@ -105,11 +106,15 @@ In order to map or encode the payload to a specific request body, and map the re
 
 == Response codes
 
-Redpanda Connect considers any response code between 200 and 299 inclusive to indicate a successful response, you can add more success status codes with the field `successful_on`.
+HTTP response codes in the 200-299 range indicate a successful response. You can use the <<successful_on,`successful_on`>> field to add more success status codes.
```

Comment:
Another way to convey this information is by saying 2xx. It's a more common way to http codes.

 https://sitechecker.pro/what-are-2xx-status-codes/

Reply:
Will leave for now. My reasoning is that it's just an easy way to scan the values without having to look them.